src/test/java/pl/marcin/project/tomtomgeoservice/service/GeoSerwisTest.java

```diff
@@ -0,0 +1,19 @@
+package pl.marcin.project.tomtomgeoservice.service;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import pl.marcin.project.tomtomgeoservice.geocodingmodel.AddressData;
+import pl.marcin.project.tomtomgeoservice.routingmodel.RouteType;
+import pl.marcin.project.tomtomgeoservice.routingmodel.TravelMode;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class GeoSerwisTest {
+
+
+    @Test
+    void countDistanceBetweenClients() throws Exception {
```

Comment:
it all depends what you want to test here :) 
What was your idea to test here :)? Let's start with such an open question and then we'll narrow it down what we need to do next